src/utils/system_checks.jl

```diff
@@ -59,13 +61,48 @@ function PVBusCheck(buses::Array{Bus}, generators::Array{T}) where {T<:Generator
 
     for b in buses
         if b.bustype == "PV"
-            b.number in pv_list ? continue : error("The bus ", b.number, " is declared as PV without a generator connected to it")
+            b.number in pv_list ? continue : warn("The bus ", b.number, " is declared as PV without a generator connected to it")
         else
             continue
         end
     end
 end
 
+function check_angle_limits!(anglelimits::@NT(max::Float64, min::Float64))
```

Comment:
This function should either 

1) Take a namedtuple `anglelimits` as an argument and check if it is within limits, i.e. return true or false, and not mutate the argument (not recommended).
2) Take a namedtuple `anglelimits` as an argument and create a new tuple `anglelimits` which will allows have allowable limits, and return the new `anglelimits` (preferred).

NamedTuples to my understanding are immutable, so the `!` suffix is not recommended.